packages/launchpad/src/setup/OpenBrowserList.vue

```diff
@@ -285,8 +284,4 @@ const browserStatus = computed(() => {
 
 const testingTypeIcon = computed(() => props.gql.currentTestingType === 'component' ? TestingTypeComponentIcon : TestingTypeE2EIcon)
 
-onMounted(() => {
-  props.launchIfBrowserSetInCli()
```

Comment:
I removed this function as a prop and just used in the parent component. When running the tests for `OpenBrowserList` it was failing because this prop was not passed in the tests.